README.md

```diff
@@ -5,19 +5,42 @@
 
```

Comment:
I think the goal of the readme is to present the interesting highlights / most useful functionalities first, before listing everything the package can do. Thus I think optimal organization of the readme (which is closer to what readme looked like before this restructuring) would be:

- CleanVision title
- Banner image (that @ulya-tkch is working on, illustrating the purpose of the library)
- short paragraph description of the library
- quickstart code example
- more links on how to get started
- badges 
- More description of library, eg: your the extra sentences "It currently detects issues in the raw images ..." 
- Table of issue types

@sanjanag What do you think? Why did you prefer to restructure the library as you did in this PR?

Reply:
The intention in moving the table up was to highlight all the issues that the package can detect for someone new. But I see your point how it occupies a lot of important space in the starting of readme I checked some popular repos on github and I propose these sections
- CleanVision title (everyone has a title image, but we can skip it since it's ultimately going to be merged and don't want a separate identity)
- badges (This doesn't occupy much space but makes it look more professional)
- Banner image
- Overview (description of what cleanvision does)
- Installation (while it is not a big section, I think it makes sense to have this separately from a user perspective)
- Quickstart (code example)
- Table of issue types
- Resources (tutorial notebook, example notebooks, documentation, example script, example dataset)
- Remaining sections (like contributing)

Reply:
I would push the Table of issue-types under "Getting started" resources which has critical links to docs, examples, etc.

In the future, we can have a second longer "Resources" section with miscellaneous links that goes under the Table of issue types.

Reply:
I would put the badges under the overview paragraph, they basically demarcate the above-the-fold content from the rest IMO

Reply:
I think installation can just be part of quickstart as you have it now.

Reply:
between Quickstart and Table of issue types, there should be more in-depth explanations about the library can be useful for, eg. your sentences starting with: "It currently detects issues in the raw images ..."

Reply:
FYI beware about thinking badges always make repos look more professional, the CI badge can easily say "failing", etc. which will not contribute to a professional look